### ISSUE TITLE
Bad url generated for some rest requests

### ISSUE THREAD

**Issue by hc4** [Graylog2/graylog2-server#2595]
### Problem description
I've confIgured graylog with this options:
```
rest_listen_uri = http://0.0.0.0:80/
web_listen_uri = http://0.0.0.0:80/web
rest_transport_uri = http://graylog/
```
Then some xhr request will be generated with bad url. For examle:
![image](https://cloud.githubusercontent.com/assets/17475270/17359700/7275ecba-5970-11e6-9c28-e8242792d8b7.png)

### Environment
* Graylog Version: 2.1.0-beta2

**Comment by hc4** [comment]
Fixed it with:
```
rest_listen_uri = http://0.0.0.0:80/
web_listen_uri = http://0.0.0.0:80/web
rest_transport_uri = http://127.0.0.1:80/
web_endpoint_uri =
```

It seems that port in rest_transport_uri is required

**Comment by kroepke** [comment]
fixed by #2605
